lua/dashboard/init.lua

```diff
@@ -30,6 +30,9 @@ db.hide_statusline = true
 db.hide_tabline = true
 db.hide_winbar = true
 db.session_directory = ''
+db.session_loaded_directory = ''
```

Comment:
why add this dir field. just use `session_directory` is enough?

Reply:
@glepnir you are right, I was going to do something different :smile: but I was in the middle of work so I messed up. I fixed it now. I created the new variable to store the current project name. This value will only be updated on `:SessionSave` or `:SessionLoad` and if it doesn't match with the name of the current project then we don't auto-save.